api4/cloud.go

```diff
@@ -39,7 +39,7 @@ func (api *API) InitCloud() {
 	// GET /api/v4/cloud/subscription
 	api.BaseRoutes.Cloud.Handle("/subscription", api.APISessionRequired(getSubscription)).Methods("GET")
 	api.BaseRoutes.Cloud.Handle("/subscription/invoices", api.APISessionRequired(getInvoicesForSubscription)).Methods("GET")
-	api.BaseRoutes.Cloud.Handle("/subscription/invoices/{invoice_id:[A-Za-z0-9]+}/pdf", api.APISessionRequired(getSubscriptionInvoicePDF)).Methods("GET")
+	api.BaseRoutes.Cloud.Handle("/subscription/invoices/{invoice_id:in_[A-Za-z0-9]+}/pdf", api.APISessionRequired(getSubscriptionInvoicePDF)).Methods("GET")
```

Comment:
This change was intentional although I didn't notice the inability for regular invoices to be fetched when I made it 

The in_ prefix was removed because yearly subscriptions have an "upcoming" invoice that doesn't have an ID matching the "in_" pattern. 

I think if you just bump the _ inside the [] and remove the "in" prefix it should work for both?